src/initializers/kafka/index.ts

```diff
@@ -20,3 +20,5 @@ export const getKafka = () => {
   }
   return kafka;
 };
+
+export { Kafka };
```

Comment:
This was used as typings. Exporting it might actually import the module in the produced .js file.  From what I see in L10 we didn't want that